pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -96,11 +111,31 @@ public void validate(TokenCredentials credentials) {
         }
     }
 
-    public String getSecret() {
-        return secret;
+    /**
+     * @since 1.8.2
+     */
+    public String getSigningSecret() {
+        return signingSecret;
     }
 
-    public void setSecret(String secret) {
```

Comment:
I would keep the `setSecret` at least and make it set both the signing and encryption secrets.